Dalsi rozsireni obrazovek

jaxxer opened the pull request:
Dale uz ale nelze testovat na Arduino UNO kvuli male pameti SRAM.

Commit:
Pridany dalsi ramce (nastaveni casu, ...)

RTC zmeneno na fake RTC (kvuli nodostupnosti RTC modulu)

Commit:
Dalsi rozsireni obrazovek

Dale uz ale nelze testovat na Arduino UNO kvuli male pameti SRAM.

pbalazi commented:
Ok, zítra už se na to chci vrhnout

Dne út 28. 8. 2018 23:20 uživatel jaxxer <notifications@github.com> napsal:

> Dale uz ale nelze testovat na Arduino UNO kvuli male pameti SRAM.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/pbalazi/uvLampManager/pull/3
> Commit Summary
>
>    - Pridany dalsi ramce (nastaveni casu, ...)
>    - Dalsi rozsireni obrazovek
>
> File Changes
>
>    - *A* uvLampArduino_v3/moje_DS3231.h
>    <https://github.com/pbalazi/uvLampManager/pull/3/files#diff-0> (160)
>    - *M* uvLampArduino_v3/uvLampArduino_v3.ino
>    <https://github.com/pbalazi/uvLampManager/pull/3/files#diff-1> (311)
>    - *M* uvLampArduino_v3/uvLampConfig.h
>    <https://github.com/pbalazi/uvLampManager/pull/3/files#diff-2> (65)
>
> Patch Links:
>
>    - https://github.com/pbalazi/uvLampManager/pull/3.patch
>    - https://github.com/pbalazi/uvLampManager/pull/3.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pbalazi/uvLampManager/pull/3>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHnS5Opp6kOwwaJyrUmlciOvrBKViesuks5uVbQugaJpZM4WQaqU>
> .
>

Commit:
Presun inicializace ramcu do separatnihi souboru

Inicializace struktur pro ramce byla presunuta do jineho souboru. Dale
doslo k optimalizaci funkci a snizeni duplicitniho kodu.

jaxxer commented:
Inicializace ramcu presunuta do zvlastniho souboru (uz neni v setup())
